hal/src/rtl872x/spi_hal.cpp

```diff
@@ -193,6 +194,13 @@ class Spi {
         }
         os_thread_scheduling(true, nullptr);
 
+        // Enable SPI Clock
```

Comment:
Any reason we enable the clocks in `init()` now instead of `begin()`?

Reply:
Just in case. There is operation to the SPI peripheral in the `init()`